src/redux/actions/user/index.js

```diff
@@ -17,7 +17,7 @@ export const fetchUser = username => async (dispatch) => {
     const response = await instance.get(url);
     dispatch(isSuccess(response.data, GET_USER));
   } catch (error) {
-    notify.show('an error occurred, reload the page', 'error', 3000);
+    notify.show('An error occurred with your internet connection, please fix it and try reloading the page.', 'error', 3000);
```

Comment:
This code is duplicated. It can be refactored out.

Reply:
Check code at lines `20`, `30`, `42` plus `22 ` and `53` in `addBulkConcepts/index.js`. It is the same code.

Reply:
@collinewait such refactoring would require its own ticket.

Reply:
I think it doesn't need a ticket. It can be done within this ticket.

Reply:
@collinewait the ticket was to change the messages and not refactor...